configure.ac

```diff
@@ -245,7 +244,6 @@ AC_SUBST([CMAKE_INSTALL_FULL_LIBDIR], ["$libdir"])  # FIXME: need to recursively
 AC_SUBST([CMAKE_INSTALL_FULL_INCLUDEDIR], ["$includedir"])
 AC_SUBST([BLAS_LIBRARIES], ["$BLAS_LIBS"])  # FIXME: need a filename, not a -l flag
 AC_SUBST([LAPACK_LIBRARIES], ["$LAPACK_LIBS"])
-AC_SUBST([EIGEN3_INCLUDE_DIR], [`echo "$EIGEN3_CFLAGS" | sed -e s/-I//`])
```

Comment:
Fine to me if this does not break fix for #252: @yurivict : could you confirm this is still OK at your side ?